### PR TITLE
ATOMICS: renamed bitwise atomics calls

### PR DESCRIPTION
**Pull request by hoopoepg**
Signed-off-by: Sergey Oblomov <sergeyo@mellanox.com>

### CODE REVIEW

**Comment by hoopoepg** [comment]
@brminich could you look?

File: verifier/atomic/osh_atomic.c [hunk 1]
```diff
@@ -38,24 +38,24 @@ const TE_NODE atomic_tcs[] =
...
-    { NULL, osh_atomic_tc37, "longlong_for",   aopt_set_string( "tc37" ), 	"shmem_longlong_for(): A simple test.",    TEST_LONGLONG_FOR},
-    { NULL, osh_atomic_tc38, "int_or",         aopt_set_string( "tc38" ), 	"shmem_int_or(): A simple test.",          TEST_INT_OR},
-    { NULL, osh_atomic_tc39, "long_or",        aopt_set_string( "tc39" ), 	"shmem_long_or(): A simple test.",         TEST_LONG_OR},
-    { NULL, osh_atomic_tc40, "longlong_or",    aopt_set_string( "tc40" ), 	"shmem_longlong_or(): A simple test.",     TEST_LONGLONG_OR},
+    { NULL, osh_atomic_tc23, "int_xor",        aopt_set_string( "tc23" ), 	"shmem_atomic_uint_xor(): A simple test.",         TEST_INT_XOR},
+    { NULL, osh_atomic_tc24, "long_xor",       aopt_set_string( "tc24" ), 	"shmem_atomic_ulong_xor(): A simple test.",        TEST_LONG_XOR},
+    { NULL, osh_atomic_tc25, "longlong_xor",   aopt_set_string( "tc25" ), 	"shmem_atomic_ulonglong_xor(): A simple test.",    TEST_LONGLONG_XOR},
+    { NULL, osh_atomic_tc26, "int_fxor",       aopt_set_string( "tc26" ), 	"shmem_atomic_uint_fxor(): A simple test.",        TEST_INT_FXOR},
```
[the remark this answers is not in the document]
Comment (hoopoepg) [reply verifier/atomic/osh_atomic.c]: right, fixed

thank you

**Comment by hoopoepg** [comment]
@jdinan done, routines are renamed

File: verifier/atomic/osh_atomic.c [hunk 2]
```diff
@@ -38,24 +38,24 @@ const TE_NODE atomic_tcs[] =
...
+    { NULL, osh_atomic_tc23, "int_xor",             aopt_set_string( "tc23" ), 	"shmem_atomic_uint_xor(): A simple test.",         TEST_INT_XOR},
+    { NULL, osh_atomic_tc24, "long_xor",            aopt_set_string( "tc24" ), 	"shmem_atomic_ulong_xor(): A simple test.",        TEST_LONG_XOR},
+    { NULL, osh_atomic_tc25, "longlong_xor",        aopt_set_string( "tc25" ), 	"shmem_atomic_ulonglong_xor(): A simple test.",    TEST_LONGLONG_XOR},
+    { NULL, osh_atomic_tc26, "int_fetch_xor",       aopt_set_string( "tc26" ), 	"uatomic_uint_fetch_xor(): A simple test.",        TEST_INT_FETCH_XOR},
+    { NULL, osh_atomic_tc27, "long_fetch_xor",      aopt_set_string( "tc27" ), 	"uatomic_ulong_fetch_xor(): A simple test.",       TEST_LONG_FETCH_XOR},
+    { NULL, osh_atomic_tc28, "longlong_fetch_xor",  aopt_set_string( "tc28" ), 	"uatomic_ulonglong_fetch_xor(): A simple test.",   TEST_LONGLONG_FETCH_XOR},
+    { NULL, osh_atomic_tc29, "int_fetch_and",       aopt_set_string( "tc29" ), 	"uatomic_uint_fetch_and(): A simple test.",        TEST_INT_FETCH_AND},
```
Comment (brminich) [inline verifier/atomic/osh_atomic.c]: u->shmem_

**Comment by hoopoepg** [comment]
@brminich more comments?

**Pull request merged**